view/frontend/web/js/view/payment/method-renderer/omise-offsite-installment-method.js

```diff
@@ -107,9 +206,28 @@ define(
              */
             getInstallmentTerms(id) {
                 const installmentBackends = checkoutConfig.installment_backends;
+                const templateLabel = $.mage.__('%terms months (%amount THB / month)');
```

Comment:
Should we be following the standard currency amount formatting for magento?

Reply:
example please..

Reply:
I don't know how this is handled in Magento 2. Magento 1 certainly has functions for this

Reply:
well.. I don't format currency here, it is JS.
I display just simply amount without decimals as number minimum is 300.

Reply:
what od you use in Mag 1?

Reply:
`Mage::app()->getLocale()->currency($currencyCode)->toCurrency($amount)`

Reply:
for now, those limits are fixed, pirces are counted by fixed results, it has to be thai baht. getting that from magento, would bring confusion or wrong data.
if user will switch price to display it ie in GBP, this will still show data according to THB. we need to force users to see THB here. (not sure if i described that understandebly)

Reply:
however, found in mag 2 from js that there are simillar mechanisms to mag 1 to display currencies

Reply:
ok. Worth using it?

Reply:
not in this case.

Reply:
it must be THB always..

Reply:
In that case, I think we better use the same Magento format?
<img width="1440" alt="Screen Shot 2562-05-24 at 22 07 17" src="https://user-images.githubusercontent.com/2154669/58337808-cc44ab00-7e70-11e9-8171-0fa455937031.png">

`฿36,999.94 / month`
Or even better if we can use a formatter function provided by Magento, cuz there is a chance that merchant modified their store in a different format from default (i.e. some store uses `฿36,999.94 Baht / month`, some uses `36,999.94 Baht / month`.

These options are protected by `isActive()` function anyway so it shouldn't be any confusion?

Reply:
Just in case, the translation part could be `"%terms months (%amount / month)", "%terms เดือน (%amount / เดือน)"`

Reply:
i translated it following info from Carol. Is that not correct?

Reply:
The translation was corrected. Just I'm suggesting the idea that we can remove `THB` out from the translation and leave it to Magento's formatter function to handle it.
As currency format can be various depending on each store.
- `฿36,999.94 / month`
- `฿36,999.94 Baht / month`
- `36,999.94 Baht / month`


Original: `"%terms months (%amount THB / month)", "%terms เดือน (%amount บาท/เดือน)"`
If we format the currency code by using Magento formatter then those `%amount THB / month`, `%amount บาท/เดือน` can become `%amount / month`, `%amount / เดือน`.

Reply:
This currency formatting issue is still not resolved?